CONTRIBUTING.md

```diff
@@ -0,0 +1,39 @@
+# Contributing
+
+Contributions are welcome! If you want to know where to start,
+look at the [issues](https://github.com/EngoEngine/engo/issues) on github.
+
+We are always looking for more documentation and better test coverage.
+
+## Guidelines
+
+Make your changes, and submit a pull request.
+
+### Git Commit
+
+Every commit should have a clear commit message explaining what was
+changed.  Please make this informative, including why it was changed.
+Also, try to keep the first line of your message under 50 characters.
+This makes it easier to read the summary of changes in various tools.
+
+Please don't leave 'WIP' (work in progress) commits in your final pull
+request: [squash](http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html)
+this so you have a clear story in your development.
+
+### Comments and Documentation
+
+Every PR should have godoc style comments on ever piece of code modified,
+regardless of whether or not it had documentation before.  Please leave
```

Comment:
`Please leave the code better you found it`. Grammar mistake and I'd suggest we just remove this line.